static/main.html

```diff
@@ -1,6 +1,9 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <!-- uncomment this to add the react debugger:
```

Comment:
we maybe want to add the react-devtools rather in our webpack configuration than here.
https://github.com/facebook/react-devtools/tree/master/packages/react-devtools